pinot-core/src/main/java/org/apache/pinot/core/util/QueryOptionsUtils.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.util;
+
+import java.util.Map;
+
+import static org.apache.pinot.common.utils.CommonConstants.Broker.Request.QueryOptionKey;
+
+
+/**
+ * Helper methods for reading query options
+ */
+public final class QueryOptionsUtils {
```

Comment:
Any specific reason for having a utils class as opposed to a QueryOptions class, that is generated from Map<String, String> queryOptions?

Reply:
Methods were moved from GroupByUtils, hence ended up putting them in another Utils.
On second thought, I took your suggestion and make it into a QueryOptions class.